ext/opcache/ZendAccelerator.c

```diff
@@ -2841,39 +2841,10 @@ static void zps_startup_failure(const char *reason, const char *api_reason, int
 	zend_llist_del_element(&zend_extensions, NULL, (int (*)(void *, void *))cb);
 }
 
-static inline zend_result accel_find_sapi(void)
+static inline bool accel_sapi_is_cli(void)
```

Comment:
```suggestion
static inline bool disable_accel_for_sapi(void)
```

or something like that, considering that it's not only cli sapi but also phpdbg.